Doppler.Currency/Dtos/TrmResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Doppler.Currency.Dtos
+{
+    public class TrmResponse
+    {
+        public decimal Valor { get; set; }
```

Comment:
It is should be in Spanish?

Reply:
the Trm API return the properties in Spanish for it reason I created the properties in Spanish.